### ISSUE TITLE
[FEATURE REQUEST] Lightning "Alarm" for finished job

### ISSUE THREAD

**Issue by nicoportorico** [prusa3d/Prusa-Firmware#3259]
Hi everybody , in Order to keep tracking the job of many printers by looking at a "panoramic" camera , and spare some money (without buying raspberryes/arduino boards and such) , wouldn t it be nice to have a simple LED (right on the uppest side of the frame) that turns on once the print is done ? 

I think that s a really cheap but usefull feature to have .

**Comment by DRracer** [comment]
@nicoportorico even though your request will probably not make it into production, you can still make a DYI solution. AFAIK there is a G-code for switching a pin on the CPU - you can connect an LED to it and turn it on when needed in your G-code file.

```
M42 - Set pin state M42: Switch I/O pin
Usage

M42 [ P | S ]

Parameters

    P - Pin number.
    S - Pin value. If the pin is analog, values are from 0 to 255. If the pin is digital, values are from 0 to 1.
```

**Comment by wavexx** [comment]
One odd alternative would be to flash the LCD backlight for printers that have it?

**Comment by JohnnyDeer** [comment]
I´m closing this issue because there are no responses.
